### PR TITLE
Gh 64 in range

### PR DESCRIPTION
**Pull request by p013570**

### CODE REVIEW

**Comment by p013570** [comment]
Closing to reduce the number of fields and add a enum to choose the time unit.

File: core/src/main/java/uk/gov/gchq/koryphe/impl/predicate/range/AbstractInTimeRange.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import uk.gov.gchq.koryphe.predicate.KoryphePredicate;
+import uk.gov.gchq.koryphe.util.TimeUnit;
+
+/**
+ * <p>
+ * An <code>InRange</code> is a {@link java.util.function.Predicate}
```
Comment (t616178) [inline core/src/main/java/uk/gov/gchq/koryphe/impl/predicate/range/AbstractInTimeRange.java]: Does this need to be updated?

Comment (p013570) [reply core/src/main/java/uk/gov/gchq/koryphe/impl/predicate/range/AbstractInTimeRange.java]: good spot thanks!

**Comment by p013570** [comment]
I'll add some examples into Gaffer Doc: https://github.com/gchq/gaffer-doc/issues/65

**Pull request merged**